### ISSUE TITLE
Document RWD Data setup instructions for macOS

### ISSUE THREAD

**Issue by kellyi** [WikiWatershed/model-my-watershed#1714]
We should document the setup instructions for using RWD data on macOS so that it's easier to do.

It's probably worth considering, at least, adding a script or a script option which will set RWD Data up correctly each time we load the project.

**Comment by kdeloach** [comment]
Agreed. The documentation could be better.

There are some hints in the [rapid-watershed-delination README](https://github.com/WikiWatershed/rapid-watershed-delineation#environment-variables). On my workstation, I define the `RWD_DATA` env var in  `.bashrc` which points to the correct folder on my passport.

**Comment by arottersman** [comment]
Part of this, more applicable to the rwd repo itself, should be mandating the use of Docker For Mac if you're on macOS. The old Docker Toolbox is just too troublesome (as I discovered today)

**Comment by kellyi** [comment]
Note on this:

In addition to updating the Vagrantfile to add the path to rwd data here:

https://github.com/WikiWatershed/model-my-watershed/blob/develop/Vagrantfile#L94

... then reprovisioning the Worker VM, it seems like there also needs to be some update either inside the Worker VM, the RWD Docker container, or both to get the new path to take hold.

I just set up the data to work on #1767 and simply reprovisioning wouldn't work. I ended up destroying and rebuilding the worker VM and it would work then, but that loop obviously takes longer than it would if we could simply update.

I think we should consider updating the README with instructions and, once they've been found, setting up some kind of script or script option which would make the minimal necessary adjustments to the VM. Conceivably this could just involve setting some mapped environment variables; ideally it would just involve running a single command which takes the path to the data as a var like

```
./setuprwd /Volumes/kelly-passport/rwd-nhd
```

which would then immediately work on any machine without having to rebuild or reprovision anything.

**Comment by kellyi** [comment]
Another note: since I think we often do development without RWD unless we're working on an RWD-specific task, if we do set up a script I think it should also ensure that the env will still work if the data has been un-linked (since an external drive has been removed, etc).

**Comment by kdeloach** [comment]
I don't know if it's related to the bug you experienced, but I often have to manually restart the `mmw-app` docker service in the VM after reloading/reprovisioning. Sometimes, the docker service starts before Vagrant is able to mount the `RWD_DATA` folder.

**Comment by kellyi** [comment]
Just set this up without destroying the worker VM and these were the steps:

First, I updated the worker synced RWD folder in the Vagrant file (this can be done via env var too).

Then I ran:

```sh
vagrant reload worker --provision
vagrant ssh worker
sudo docker restart $(sudo docker ps -n=1 -q)
```

That should get and restart the RWD container so that the rwd data path takes hold; `-n=1` grabs only the most recently created container, which seems to be RWD. The worker's also got a spark-job-server container running, but I think the Ansible rules create SJS first then RWD.

You can find the RWD container ID for it by running this in Vagrant:

```sh
sudo docker ps | grep rwd
```

**Comment by arottersman** [comment]
Will `sudo docker restart rwd` work?

**Comment by kdeloach** [comment]
The upstart job exists to handle starting and stopping the service. Although, I have noticed that doing `restart` is not as reliable as doing `stop` followed by `start`.

```
sudo stop mmw-rwd
sudo start mmw-rwd
```

**Comment by kellyi** [comment]
> Will sudo docker restart rwd work?

Yeah, this seems to work too!, as do the `sudo stop mmw-rwd`/`sudo start mmw-rwd` commands. We should add the latter to the README somewhere, as I think those would be the best ones to use. Looks like we should also note that the Worker VM containers the RWD service in a Docker container.

**Comment by kdeloach** [comment]
Thanks for figuring this stuff out. I should have documented these quirks.

**Comment by lewfish** [comment]
You should also make sure you have the latest version of the data when setting up the project.
